pynemo/nemo_bdy_extr_tm3.py

```diff
@@ -869,19 +967,32 @@ def write_out(self, year, month, ind, unit_origin):
         
         # Loop over variables in extracted object
             
-#        for v in self.variables:
-        for v in self.var_nam:
-            if self.settings['dyn2d']: # Calculate depth averaged velocity
+        #        for v in self.variables:
+        if self.key_vec == True:
+            if self.rot_dir == 'i':
+                varnams = [self.var_nam[0],]
+            else:
+                varnams = [self.var_nam[1],]
```

Comment:
as both source u and v are passed, use rot_dir to determine whether we're calculating destination u or destination v